pandas/tests/series/test_ufunc.py

```diff
@@ -349,12 +349,14 @@ def test_max(self, values_for_np_reduce, box_with_array):
             tm.assert_series_equal(result, expected)
         else:
             expected = values[1]
+            if using_python_scalars and values.dtype.kind in ["i", "f"]:
```

Comment:
NBD but checking for `kind in "if"` is very slightly faster than checking for `kind in ["i", "f"]`.